src/drt/src/TritonRoute.tcl

```diff
@@ -58,7 +58,7 @@ sta::define_cmd_args "detailed_route" {
     [-no_pin_access]
     [-min_access_points count]
     [-save_guide_updates]
-    [-repair_pdn_vias layer]
+    [-repair_pdn_vias begin_layer end_layer]
```

Comment:
This will require updating ORFS and the private PDKs.  Is it possible to accept either a single layer or a pair?

Reply:
We could modify the detail_route script to replicate the same layer as the begin and end layer instead of doing it from the router.

Reply:
@maliberty How should I proceed with that solution? Should I open a PR from ORFS? which should be merged first?